domains/miaomingc.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "miaomingc",
+        "email" : "miaomingc@outlook.com"
+    },
+    "record": {
+        "A": "204.194.66.196",
+        "AAAA":"2001:470:18:c89::2"
```

A and AAAA records should be in an array
```suggestion
        "A": ["204.194.66.196"],
        "AAAA": ["2001:470:18:c89::2"]
```